packages/design-system/src/components/Icon/DeprecatedIconHelper.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { ReactElement } from 'react';
+import { Icon } from './Icon';
+import { SizedIcon } from './SizedIcon';
+import { IconNameWithSize } from '@talend/icons';
+
+export function parseDeprecatedIcon({
```

Comment:
A helper function that helps decide whether something should be a new icon or a deprecated one, at runtime. 

When (if 😭 ) the old icons finally die their deserved death, this function could be easily tracked down and replaced with `SizedIcon` imports.

Reply:
Just in case, the name of this function looks a bit weird for me.
I don't understand that we get an icon returned.
Wdyt about something like `getDeprecatedIcon` (not 100% convinced anyway but still)

Reply:
`getIconWithDeprecatedSupport` is my suggestion 🤷